lib/transports/idbTransport.js

```diff
@@ -28,6 +28,11 @@ try {
 }
 
 function idbCall(config, xmlInput, cb) {
+  // idb transport is not available bail out
+  if (idb === null) {
+    cb(new Error('idb-connector module was not found, the idb transport is only usable on IBM i.'), null);
```

Comment:
Maybe something like 

> "idb-connector was not found, ensure idb-connector is installed properly (idb-connector is only available on IBM i)"

I get nervous about the term "module", since idb-connector doesn't have `type: module` in its package.json, which affects all sorts of things (CJS vs. ESM modules)

Reply:
Sounds good to me!

Reply:
Fixed up in: https://github.com/IBM/nodejs-itoolkit/pull/353/commits/12083c93641326230979ab87bc3512515a375955